### DISCUSSION TITLE
In some cases dischtime of previous admission is later than admittime of new admission.

### DISCUSSION THREAD

**Discussion by kalrashid15** [MIT-LCP/mimic-code#1701] [MIMIC-IV] [answered]
There are 59 such incidences in admissions.csv. Where, same patients with multiple admissions have their previous discharge time succeeds their admission time of the following admission.

**Comment by tompollard** [comment, marked as answer]
This post is a duplicate of https://github.com/MIT-LCP/mimic-code/discussions/1702?converting=1 (converted from issue to discussion). I'll close this discussion and leave the other one up, because (1) the other post came first (2) the other description does more to demonstrate the issue. Short answer, I think, is that sometimes there are errors in the source data. We do not try to fix them.
